<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choixParcoursMenuClass</class>
 <widget class="QWidget" name="choixParcoursMenuClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_Back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Titre">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Parcours</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_Parcours">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>571</width>
     <height>211</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Num</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nom</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Difficult√©</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>test</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>test</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>test</string>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>test</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>test2</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>test2</string>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string>test2</string>
    </property>
   </item>
   <item row="1" column="3">
    <property name="text">
     <string>test2</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_Valider">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
